test/libraries/Hooks.t.sol

```diff
@@ -943,12 +943,6 @@ contract HooksTest is Test, Deployers, GasSnapshot {
         );
     }
 
-    function test_gas_hookShouldCallBeforeSwap() public {
-        snapStart("HooksShouldCallBeforeSwap");
```

Comment:
why are these gas tests being removed? 

and can you clear them out of .forge-snapshots/ if its necessary they are removed?

Reply:
they were cleared from ./forge-snapshots already, these tests are removed because they cause an underflow because via ir in the forge gas snapshot library with the gas calibration variable. These functions only measure gas, so there is no real test coverage lost in removing them